src/lib/utils/thorchain/memo.ts

```diff
@@ -0,0 +1,136 @@
+import { bn } from 'lib/bignumber/bignumber'
+
+import { THORCHAIN_AFFILIATE_NAME } from './constants'
+
+const assertIsValidPool = (pool: string, memo: string) => {
```

Comment:
The name of this might be misleading - it's just checking that a pool exists (truthy), not that it's valid.

Reply:
570726bea1